### DISCUSSION TITLE
Line breaks and negations in Labels?

### DISCUSSION THREAD

**Discussion by MoDimGH** [hneemann/Digital#975] [Q&A] [answered]
Hi, is there possibly a way to have line breaks within text labels so one can write in a new line? And also can an input label have a dash above its text characters to insinuate it is a null-active input?
I sadly could not find it :(

**Comment by xotmatrix** [comment, marked as answer]
You can create a overbar complement / active-low label by preceding it with a tilde.
`~CE`
If you need a bar over more than one word, enclose them in braces.
`~{Chip Enable}`

I'm unaware of a way to add line breaks / newlines to a label.

**Reply by MoDimGH** [reply]
Thanks that's very helpful
